src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -334,14 +344,51 @@ export function addDevEnvConfigurationOptions( command ) {
 		.option( 'media-redirect-domain', 'Domain to redirect for missing media files. This can be used to still have images without the need to import them locally.' );
 }
 
-function getWordpressImageTags(): string[] {
-	return [
-		'5.9',
-		'5.8.3',
-		'5.8.2',
-		'5.8.1',
-		'5.8',
-		'5.7.3',
-		'5.7.2',
-	];
+async function populateWordPressVersionList( versionList ) {
```

Comment:
1. Instead of using `https` we should use `node-fetch` (as other parts of VIP-CLI do), this will allow for uniformity and better error handling, and make the code more concise.
2. The error handling is a bit rudimentary:
  - What happens if GH  is down?
  - What happens if someone accidentally removed the list from the repo?

Reply:
Thank you for your feedback. I took some time to address the concerns you had, and refactored the code a bit.

Here are the highlights:

- I changed the https api code to use `node-fetch`. This definitely made the net code more concise.
- Because of caching, I was careful to design logic around retaining the cache file when it exists. Now, in circumstances where github is down or if someone removes the manifest from the repo, the program will continue to run using the cache; and with a small yellow message about it.
![Screen Shot 2022-02-09 at 7 19 16 AM](https://user-images.githubusercontent.com/46167019/153221208-2b1b0af6-60a2-4ec1-9d77-3d5b1f912a6a.png)